appservice/src/tree/DeploymentTreeItem.ts

```diff
@@ -78,17 +79,18 @@ export class DeploymentTreeItem extends AzureTreeItem<ISiteTreeRoot> {
     }
 
     public async redeployDeployment(): Promise<void> {
-        const redeploying: string = `Redeploying commit "${this.id}" to "${this.root.client.fullName}"`;
-        const deployed: string = `Commit "${this.id}" has been redeployed to "${this.root.client.fullName}".`;
+        const redeploying: string = localize('redeploying', 'Redeploying commit "{0}" to "{1}"... Check output window for status.', this.id, this.root.client.fullName);
```

Comment:
The "..." in the middle of these messages has always bothered me. Why isn't it just a single period?

Reply:
I thought we did it because it was an ongoing message.  I don't really have an opinion on it though.